src/main/java/com/eppo/sdk/helpers/FetchConfigurationsTask.java

```diff
@@ -16,7 +16,8 @@ public class FetchConfigurationsTask extends TimerTask {
   @Override
   public void run() {
     configurationStore.fetchAndSetExperimentConfiguration();
-    long delay = this.intervalInMillis - ((long) Math.random() * this.jitterInMillis);
+    // TODO: retry on failed fetches
```

Comment:
I figure we'll want to port the same polling retry logic that we did for JavaScript